common.mk

```diff
@@ -30,6 +30,7 @@ UNAME_M				:= $(shell uname -m)
 BUILD_PATH			?= $(ROOT)/build
 LINUX_PATH			?= $(ROOT)/linux
 UBOOT_PATH			?= $(ROOT)/u-boot
+BENCHMARK_APP_PATH	?= $(ROOT)/optee_benchmark
```

Comment:
Indentation looks incorrect (I cross checked in my editor as well).

Reply:
Fixed